streams/us_stirr.m3u

```diff
@@ -1,4 +1,16 @@
 #EXTM3U
+#EXTINF:-1 tvg-id="24HourFreeMovies.us",24 Hour Free Movies (720p)
+https://d1j2u714xk898n.cloudfront.net/v1/master/9d062541f2ff39b5c0f48b743c6411d25f62fc25/STIRR-MuxIP-24HourFreeMovies/145.m3u8
```

Comment:
Basically is just https://d1j2u714xk898n.cloudfront.net/scheduler/scheduleMaster/145.m3u8